src/_playground/documentation/Properties/_PropertyShape.js

```diff
@@ -0,0 +1,59 @@
+import Heading from '../Heading/Heading';
+import PropertyDefault from './_PropertyDefault';
+import PropertyType from './_PropertyType';
+import PropTypes from 'prop-types';
+import React from 'react';
+import sortBy from 'sort-by';
+import Table from '../../../Table/Table';
+
+const PropertyShape = ({ title, propTypes, defaultProps }) => {
+    if (!propTypes) {
+        return (
+            <em>This shape has no defined properties.</em>
+        );
+    }
+
+    let propInfo = Object.keys(propTypes).map((propName) => {
+        return { propName: propName, required: propTypes[propName].typeRequired };
+    });
+    const sortedProps = propInfo.sort(sortBy('-required', 'propName'));
+
+    let data = [];
+
+    sortedProps.forEach(({ propName }) => {
+        data.push({
+            rowData: [
+                propName,
+                <PropertyType prop={propTypes[propName]} />,
+                <PropertyDefault
+                    defaultValue={defaultProps && defaultProps[propName]}
+                    prop={propTypes[propName]} />
+            ]
+        });
+    });
+
+    return (
+        <div className='frDocs-Properties__shape'>
+            {title && <Heading level={4}>{title}</Heading>}
```

Comment:
Is there a way for us to change the title so that it says `Counter - Localized Text` instead of `Localized Text - Counter`. This is super nit picky so if anyone disagrees I am fine with this way. 😄 . I just like Counter first as its the parent item and localization is the loc text being a subset of that item.

Reply:
Makes sense to me 👍